### PR TITLE
[Docs] Update "Customizing State Machine"

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         | 1.3
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

File: docs/customization/state_machine.rst [hunk 1]
```diff
@@ -77,23 +82,27 @@ How to add a new callback?
...
             callbacks:
                 after:
-                    sylius_send_email:
+                    sylius_update_rating:
                         # here you are choosing the transition on which the action should take place - we are using the one we have created before
-                        on: ["cancel"]
-                        # it is just an example, use an existent service and its method here!
-                        do: ["@service", "sendEmail"]
+                        on: ["accept"]
+                        # use an existent service and its method here are create your own service!
```
Comment (GSadee) [inline docs/customization/state_machine.rst]: Sth is wrong with this sentence

File: docs/customization/state_machine.rst [hunk 2]
```diff
@@ -77,23 +82,27 @@ How to add a new callback?
...
+                        do: ["@App\\ProductReview\\Mailer\\ConfirmationMailer", "sendEmail"]
                         # this will be the object of an Order here
                         args: ["object"]
 
+.. tip::
+    If you wont see implementation of ``ConfirmationMailer`` check it on `this GitHub Pull Request.
```
Comment (GSadee) [inline docs/customization/state_machine.rst]: `If you want to see...`

File: docs/customization/state_machine.rst [hunk 3]
```diff
@@ -77,23 +82,27 @@ How to add a new callback?
 Let's assume that you would like to add a new **callback** to :doc:`the Order state machine </book/orders/orders>`,
 that will do something on an already defined transition.
```
Comment (CoderMaggie) [inline docs/customization/state_machine.rst]: This description is no longer relevant after you've changed the example.

File: docs/customization/state_machine.rst [hunk 4]
```diff
@@ -77,23 +82,27 @@ How to add a new callback?
...
+                        do: ["@App\\ProductReview\\Mailer\\ConfirmationMailer", "sendEmail"]
                         # this will be the object of an Order here
                         args: ["object"]
 
+.. tip::
+    If you wont see implementation of ``ConfirmationMailer`` check it on `this GitHub Pull Request.
+    <https://github.com/Sylius/Customizations/pull/20>`_
+
 After that your new callback will be available alongside other callbacks that already were defined in that state machine
 and will be called on the desired transition
```
Comment (CoderMaggie) [inline docs/customization/state_machine.rst]:
```suggestion
and will be called on the desired transition.
```

File: docs/customization/state_machine.rst [hunk 5]
```diff
@@ -74,38 +79,38 @@ How to remove a state and its transitions?
...
+    <https://github.com/Sylius/Customizations/pull/20>`_
+
+After that your new callback will be available alongside other callbacks that already were defined in that state machine and will be called on the desired transition.
 
 How to modify a callback?
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you would like to modify an existent callback of for example the state machine of ProductReviews,
-so that it does not count the average rating but does something else - you need to add these few lines to the ``config/_sylius.yaml``:
+so that it does not count the average rating but does something else - you need to add these few lines to the ``config/packages_sylius.yaml``:
```
Comment (Zales0123) [inline docs/customization/state_machine.rst]: `config/packages_sylius.yaml` -> `config/packages/_sylius.yaml`

File: docs/customization/state_machine.rst [hunk 6]
```diff
@@ -74,38 +79,41 @@ How to remove a state and its transitions?
...
+                        on: ["accept"]
+                        # use service and its method here
+                        do: ["@App\\ProductReview\\Mailer\\ConfirmationMailer", "sendEmail"]
                         # this will be the object of an Order here
                         args: ["object"]
 
-After that your new callback will be available alongside other callbacks that already were defined in that state machine
-and will be called on the desired transition
+.. tip::
+    If you want see implementation of ``ConfirmationMailer`` check it on `this GitHub Pull Request.
```
Comment (GSadee) [inline docs/customization/state_machine.rst]: ...want **to** see **the** implementation...

Comment (GSadee) [inline docs/customization/state_machine.rst]:
```suggestion
    If you want to see the implementation of ``ConfirmationMailer`` check it on `this GitHub Pull Request.
```

**Comment by GSadee** [comment]
Thank you, Adam! :tada:

**Pull request merged**